pkg/kritis/review/review.go

```diff
@@ -52,25 +85,144 @@ func (r Reviewer) Review(images []string, isps []v1beta1.ImageSecurityPolicy, po
 	}
 	for _, isp := range isps {
 		for _, image := range images {
+			glog.Infof("Check if %s as valid Attestations.", image)
+			isAttested, attestations := r.fetchAndVerifyAttestations(image, isp.Namespace, pod)
+			// Skip vulnerability check for Webhook if attestations found.
+			if isAttested && r.isWebhook {
+				continue
+			}
+
 			glog.Infof("Getting vulnz for %s", image)
-			violations, err := r.validate(isp, image, r.client)
+			violations, err := r.config.validate(isp, image, r.client)
 			if err != nil {
 				return fmt.Errorf("error validating image security policy %v", err)
 			}
 			if len(violations) != 0 {
-				errMsg := fmt.Sprintf("found violations in %s", image)
-				// Check if one of the violations is that the image is not fully qualified
-				for _, v := range violations {
-					if v.Violation == securitypolicy.UnqualifiedImageViolation {
-						errMsg = fmt.Sprintf("%s is not a fully qualified image", image)
-					}
+				return r.handleViolations(image, pod, violations)
+			} else if r.isWebhook {
```

Comment:
the above if block returns, so "else if" isn't necessary.

Also, what about if r.isWebhook is false?

Reply:
do nothing.  we have already handled violations in line 101. If there are no violations, the cron does nothing but proceeds with the next image scanning.